src/ui/Pricing/SubscriptionSettings/index.svelte

```diff
@@ -179,12 +179,65 @@
     gap: 16px;
   }
 
+  .overview {
+    margin-top: 6px;
+  }
+
+  .second-btn {
+    margin-left: 12px;
+  }
```

Comment:
Apply non responsive margins using webkit's margin classes `mrg-[size]` and `mrg--[dir]`